content/flux/v0.x/release-notes.md

```diff
@@ -10,6 +10,30 @@ aliases:
   - /influxdb/cloud/reference/release-notes/flux/
 ---
 
+## v0.172.0 [2022-06-24]
+
+### Features
+- Add multiple new join functions to the `join` package such as `join.full()`.
+- Add `initialZero` parameter to derivative function. 
+- Allow features to enable builtin statements. 
+- Provide the comments for each `Symbol` from `PackageExports`.
+- Suggestions now start off by default and added a new flag.
+- Add builtin function time to the `date` package to convert any timeable into datetime. 
+- Allow vector types to be specified in flux source. 
+
+### Bug fixes
+- Replace extra boolean parameter for suggestions with repl options. 
+- Remove `testing.load()` from yield test. 
```

Comment:
I think yield test may refer to [testutil.yield()](https://docs.influxdata.com/flux/v0.x/stdlib/internal/testutil/yield/), but lets double-check

Reply:
thanks for catching this. i checked and it does refer to testutil().yield()